source/opt/aggressive_dead_code_elim_pass.cpp

```diff
@@ -245,11 +245,11 @@ void AggressiveDCEPass::AddBranch(uint32_t labelId, BasicBlock* bp) {
 }
 
 void AggressiveDCEPass::AddBreaksAndContinuesToWorklist(
-    Instruction* loopMerge) {
-  BasicBlock* header = context()->get_instr_block(loopMerge);
+    Instruction* mergeInst) {
+  BasicBlock* header = context()->get_instr_block(mergeInst);
   uint32_t headerIndex = structured_order_index_[header];
   const uint32_t mergeId =
-      loopMerge->GetSingleWordInOperand(kLoopMergeMergeBlockIdInIdx);
+      mergeInst->GetSingleWordInOperand(0);
```

Comment:
Perhaps assert that this is, indeed, a merge instruction?  They're all guaranteed to have the merge ID in operand 0, right?

Reply:
Added the assert, and yes both merge instructions have the merge id at position 0.